Dnn.AdminExperience/ClientSide/Pages.Web/src/components/Seo/PageUrls/EditUrl.jsx

```diff
@@ -140,7 +140,10 @@ class EditUrl extends Component {
                         <Button type="secondary" onClick={onCancel} disabled={saving}>
                             {Localization.get("Cancel")}
                         </Button>
-                        <Button type="primary" onClick={onSave} disabled={!this.state.hasChanges || saving}>
+                        <Button
+                            type="primary"
+                            onClick={onSave}
+                            disabled={!this.state.hasChanges || url == null || url.siteAlias == null || url.siteAlias.Key == null || saving}>
```

Comment:
I don't believe that this check is proper, as it SHOULD be possible to set a redirect that is a 301 otherwise that applies to all site aliases.  This is what has support in the database within the TabUrls table as well as the URL Rewriting rules.

If anything this should be adjusted to validate based on type of redirect

Reply:
Hi @mitchelsellers , do you mean that Site Alias field is optional? When the user doesn't select a Site Alias and click on save button, it adds the URL for an Alias that's not selected at all (and it's not editable too). I don't think this is the desired behavior though, right? You can check this behavior in this [link](https://drive.google.com/drive/folders/1xvwctzL1dTAQwSWO6kuOo1QAlrfkUEZp) (between 0:35-0:52). I just made this change to ensure that a site alias is selected before saving the url.

@v-karbovnichy , could you also check this please?

Reply:
@berkarslan-xo No, I don't believe an alias is required, it isn't at the database level and there are MANY reasons why you wouldn't want it in there for flexibility with redirects.

@david-poindexter or @valadas any thoughts?

Reply:
Agreed.  I think the bug is that the UI creates an alias-specific URL entry when you don't select an alias.  It should allow the alias to remain `null`.

Reply:
Yeah I agree this is optional

Reply:
Got it. I'm closing this PR then.

Reply:
Agreed for what it is worth.  ;-)

Reply:
Thanks everyone for the inputs btw.